week-3/slideshow/index.html

```diff
@@ -3,12 +3,25 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Slideshow</title>
```

Comment:
Some of he images seemed squished. How could you improve to maintain their aspect ratios intact?